packages/astro/test.d.ts

```diff
@@ -0,0 +1,2 @@
+/// <reference path="./astro-jsx.d.ts" />
+export function render(vnode: astroHTML.JSX.Element): Promise<{ container: HTMLTemplateElement }>;
```

Comment:
It would be cool if this function also worked inside of .astro components, can we make this accept a non-JSX astro type? I'm thinking I'd love to be able to do:

__src/components/Greeting.test.astro__

```astro
---
import { expect, test } from 'vitest'
import { render } from 'astro/test'
import Greeting from './Greeting.astro'

test('Greeting', async () => {
	const { container } = await render(<Greeting />)
	expect(container).toBeDefined()
	expect(container.textContent).toBe('Hello world')
})
---
```

Reply:
I realize that the above probably doesn't work for some *reasons*, just noting that it would be nice if this generically named `render` function can be extended in the future for non-JSX test files.